services/gitauto_handler.py

```diff
@@ -97,6 +97,7 @@ async def handle_gitauto(payload: GitHubLabeledPayload, trigger_type: str) -> No
         "base_branch": base_branch_name,
         "new_branch": new_branch_name,
         "token": token,
+        "reviewers": list(set([sender_name, issuer_name]))
```

Comment:
I agree with Sourcely AI feedback. You can do this `list({sender_name, issuer_name})`

Reply:
@BigOleHealz Here is my tiny comment!